src/tls/Makefile

```diff
@@ -341,3 +364,10 @@ test13:
 ifeq ($(OS),Windows_NT)
     EXTRA_PATH = PATH="/usr/x86_64-w64-mingw32/sys-root/mingw/bin/:$(PATH)"
 endif
+
+# FFI support - calling from C into miTLS
+#depends on tls-gen
+tls-ffi: libmitls.dll
+
+libmitls.dll: mitls.cmxa #tls-gen
+	$(OCAML) $(OCAMLOPTS) $(OCAML_INCLUDE_PATHS) $(LIB_ML) -inline 0 -linkall -g -output-obj mitls.cmxa -o libmitls.dll
```

Comment:
Build performance - it makes quite a different in the compile time, though code quality is not nearly as good.  I'll remove.